src/main/java/com/github/junrar/crypt/Rijndael.java

```diff
@@ -63,7 +61,9 @@ public static Cipher buildDecipherer(final String password, byte[] salt) throws
 
         for (int i = 0; i < HashRounds; i++) {
             bout.write(rawpsw);
-            bout.write(new byte[] {(byte) i, (byte) (i >> 8), (byte) (i >> 16)});
+            bout.write((byte) i);
```

Comment:
On a ByteArrayOutputStream this makes no difference because internally it just iterates over the array anyway.

Reply:
So this ends up being faster/saving some object allocation